config/env_sample.hjson

```diff
@@ -252,7 +253,36 @@
                 and membership.id is not null 
                 and REGEXP_EXTRACT(`group`.id, r'.*:(.*)') IN UNNEST(@course_ids)
                 '''
-        }
+        },
+        "canvas_pages": {
+            "query":
+                '''
+                SELECT 'canvas_pages' AS resource_type,
+                CONCAT(CAST(REGEXP_EXTRACT(`group`.id, r'.*:(.*)') AS INT64) - @canvas_data_id_increment, 
+        				'/pages/', 
+        				CAST(JSON_EXTRACT_SCALAR(object.extensions, '$[\'com.instructure.canvas\'][entity_id]') as INT64) - @canvas_data_id_increment) as resource_id,
+                CAST(REGEXP_EXTRACT(membership.id, r'.*:(.*)') AS INT64) AS user_id,
+                cast(null as string) AS user_login_name,
+                CAST(REGEXP_EXTRACT(`group`.id, r'.*:(.*)') AS INT64) AS course_id,
+                JSON_EXTRACT_SCALAR(object.extensions, '$[\'com.instructure.canvas\'][asset_name]') as name,
+                datetime(EVENT_TIME) as access_time
+                FROM event_store.expanded
+                where
+                JSON_EXTRACT_SCALAR(ed_app.json, '$.id') IN UNNEST(['http://m.example.instructure.com/', 'http://example.instructure.com/'])
```

Comment:
Maybe Nitpick: We should have these example URL's match the other query on line 248 which are actually umich.edu ones. 

We also should consider a new issue to passing these in from a configuration parameter since they're used in 2 queries now. Then the query would be able to be copied and pasted without any changes. 

We also don't need the JSON_EXTRACT_SCALAR here anymore and can just change this to

`                ed_app.id IN UNNEST(['http://m.canvas.umich.edu/', 'http://umich.instructure.com/']) 
`

Reply:
Those url values are now umich-related. Also created #1397 to parametrize those values